src/OrchardCore/OrchardCore.Abstractions/Json/Serialization/DateTimeJsonConverter.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace OrchardCore.Json.Serialization;
+
+public class DateTimeJsonConverter : JsonConverter<DateTime>
+{
+    public override DateTime Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+    {
+        Debug.Assert(typeToConvert == typeof(DateTime));
```

Comment:
```suggestion
        if(typeToConvert is not typeof(DateTime))
        {
            throw new ArgumentException("Unexpected type to convert.", nameof(typeToConvert));
        }
```

Reply:
Check this @hyzx86

Reply:
Done

Reply:
Seems the build failed, that's why I told you check, coz I'm working on another PR

Reply:
Just a moment, I think it's necessary to add a validation test in the wrong date time format.